runtime/src/main/java/org/corfudb/runtime/LogReplicationUtils.java

```diff
@@ -146,8 +162,13 @@ public Void call() {
             Optional<Counter> conflictCounter = MicroMeterUtils.counter("logreplication.subscribe.conflict.count");
             Optional<Timer.Sample> subscribeTimer = MicroMeterUtils.startTimer();
 
-            Table<LogReplicationSession, ReplicationStatus, Message> replicationStatusTable =
-                    openReplicationStatusTable(corfuStore);
+            try {
+                replicationStatusTable = corfuStore.getTable(CORFU_SYSTEM_NAMESPACE, REPLICATION_STATUS_TABLE_NAME);
```

Comment:
I see that we are now avoiding open() if the table was already opened.  But unable to follow how it was causing an error.

Reply:
The error was actually due not having [this](https://github.com/CorfuDB/CorfuDB/pull/3799/files#diff-0a0378e999458fc223446d6ee8d025ee06b22635241f6c5541cc51e49fb9ab79R63) outside of the full sync task, which would not guaranteed the status table being opened in case that task was halted for any reason.

Reply:
actually I'm a bit confused by the IRetry for openTable operation, I saw @hisundar 's comment, but the txn that open table will be doing is just writing to RegistryTable, if we check the `registerTable` method in `TableRegistry`, it already has its own retry logic. So @hisundar  please correct me if I missed anything

Reply:
so IMO the try - getTable - catch - openTable pattern you've been using in LogicalGroupClient should be fine for that helper method